testing/endpoint.go

```diff
@@ -492,6 +493,85 @@ func (endpoint *Endpoint) TimeoutOnClose(packet channeltypes.Packet) error {
 	return endpoint.Chain.sendMsgs(timeoutOnCloseMsg)
 }
 
+// UpgradeChain performs a IBC client upgrade using the provided client state.
+// The chainID within the client state will have its revision number incremented by 1.
+// The counterparty client will be upgraded if it exists.
+func (endpoint *Endpoint) UpgradeChain(clientState *ibctmtypes.ClientState) error {
```

Comment:
It might be useful to pass revisionID as a parameter rather than incrementing :thinking: